src/Microsoft.IdentityModel.Tokens/Validation/ValidationParameters.cs

```diff
@@ -494,9 +550,16 @@ public string RoleClaimType
         /// Gets the <see cref="IList{String}"/> that contains valid types that will be used to check against the JWT header's 'typ' claim.
         /// If this property is not set, the 'typ' header claim will not be validated and all types will be accepted.
         /// In the case of a JWE, this property will ONLY apply to the inner token header.
-        /// The default is <c>null</c>.
+        /// The default is an empty collection.
         /// </summary>
-        public IList<string> ValidTypes { get; }
+        /// <exception cref="ArgumentNullException">Thrown when the value is set as null.</exception>
+        /// <returns>The <see cref="IList{String}"/> that contains valid token types that will be used to check against the token's 'typ' claim.</returns>
+        public IList<string> ValidTypes
+        {
+            get => _validTokenTypes ?? Interlocked.CompareExchange(ref _validTokenTypes, new List<string>(), null);
```

Comment:
`_validTokenTypes` is never null, as you are assigning `[]` at the beginning of the file and throwing if someone attempts to assign null. This could be simplified.

Reply:
Updated. Thank you.